prime-router/docs/proposals/0017-metadata-tables-cleanup/elr-metadata-table-changes.sql

```diff
@@ -0,0 +1,381 @@
+    , state_fips_code           varchar(50) null
+    , country_id                bigint not null references country(country_id)
+);
+
+comment on table state is
+    'A table of states for the messages we get. These should correspond to the USPS values for States, when dealing with US states.';
+comment on column state.state_id is
+    'The primary key for the state table';
+comment on column state.state_name
+    is 'The common name for the US state, territory, or province. For example, use Pennsylvania instead of The Commonwealth of Pennsylvania';
+comment on column state.state_abbreviation is
+    'The two-letter abbreviation used by the US Postal Service, or the correct abbreviation for Canadian addresses';
+comment on column state.state_fips_code is
+    'The FIPS numeric code used for the state. In the case of Canadian provinces, this would be null.';
+comment on column state.country_id is
+    'The foreign key for the country the state is in';
+
+
+-- a county table
+create table county
+(
+    county_id                   bigserial not null primary key
+    , county_name               varchar(512) not null
+    , county_fips_code          varchar(50) null
+    , state_id                  bigint not null references state(state_id)
+);
+
+comment on table county is
+    'A table for the county an address can be in.';
+comment on column county.county_id is
+    'The primary key for the county table';
+comment on column county.county_name is
+    'The name of the county';
+comment on column county.county_fips_code is
+    'The numeric code for the county per FIPS';
+comment on column county.state_id is
+    'The foreign key for the state a county is in. Per the Census Bureau no county can cross state lines';
+
+
+-- a facility could exist in multiple roles: ordering facility, testing facility,
+-- but in the future it could also be a servicing facility for vaccines or patient care
+-- at a hospital
+create table facility_role
+(
+    facility_role_id                    bigserial not null primary key
+    , facility_role                     varchar(512) not null
+);
+
+comment on table facility_role is
+    'The different kinds of roles a facility can play, for example an ordering facility, a reporting facility, etc';
+comment on column facility_role.facility_role_id is
+    'The primary key for the facility roles';
+comment on column facility_role.facility_role is
+    'The type of role a facility can play';
+
+-- insert the first few rows
+insert into facility_role (facility_role) values ('ordering');
+insert into facility_role (facility_role) values ('testing');
+insert into facility_role (facility_role) values ('reporting');
+
+
+-- a unified list of facilities
+-- TODO: should we separate out the address from the facility? can a facility have more than one address?
+create table facility
+(
+    facility_id                         bigserial not null primary key
+    , facility_name                     varchar(512) not null
```

Comment:
We've had one issue in the past where facility names were too long, and character number allowed was a lot lower than 512.